2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -103,6 +119,7 @@ collector {
     }
 
     # Incoming events are stored in a buffer before being sent to Kinesis/Kafka.
+    # Note: Buffering is not supported by NSQ; will be ignored    
```

Comment:
Can't we use `byte-limit` as [output buffer size](https://github.com/brainlag/JavaNSQClient/blob/master/src/main/java/com/github/brainlag/nsq/NSQConfig.java#L76-L78) and `time-limit` as [output buffer timeout](https://github.com/brainlag/JavaNSQClient/blob/master/src/main/java/com/github/brainlag/nsq/NSQConfig.java#L84-L86)?